conftest.py

```diff
@@ -1,17 +1,44 @@
-import logging
-
+# -*- coding: utf-8 -*-
 import pytest
 
+try:
+    # TODO: this file is read/executed even when called from ``readthedocsinc``,
+    # so it's overriding the options that we are defining in the ``conftest.py``
+    # from the corporate site. We need to find a better way to avoid this.
```

Comment:
`conftest.py` is directory dependent. That's why we can't override these pytest's options here. I'd like to find a better way to do this.

Reply:
Does it make sense to define these `PYTEST_OPTIONS` in the `settings.py` file?

Reply:
Perhaps this would be a good addition. Alternatively, maybe something in setup.cfg instead? I'm not sure if the load order would allow us to use django settings, given pytest-django likely sets up the application later in the process. We can make an issue to track updating this pattern.

Reply:
I opened #4317 for this